### ISSUE TITLE
[Component Update] `control item` based components must have hidden divider by default

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/ouds-flutter#379]
### Description

For _checkbox_, with version 2.2.0 of Figma specifications, in the initial settings, the _divider_ variant is now hidden.

Same thing for _radio_ with version 1.2.0 and and _switch_ with version 1.3.0

For example, bellow are udpated specifications for _checkboxes_.

<img width="834" height="135" alt="Image" src="https://github.com/user-attachments/assets/9e460734-5fad-48d5-b854-4f039d54f793" />

<img width="834" height="273" alt="Image" src="https://github.com/user-attachments/assets/24404f20-9807-47b9-ab39-bcb52669405c" />

**Comment by B3nz01d** [comment]
@AhmedAmineZr which alpha version should be used to perform this test?

**Comment by AhmedAmineZr** [comment]
@B3nz01d  this ticket is finished on dev but we need to update Token to can delivery

**Comment by AhmedAmineZr** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (alpha) (444)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 379
cc @B3nz01d (product owner)
cc @Tayebsed93 @nouha06 @AhmedAmineZr (dev team)
cc @jerome-regnier (design team)
cc @pya35 (Accessibility)

**Comment by AhmedAmineZr** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (444)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 379
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (Accessibility)

**Comment by B3nz01d** [comment]
closed during sprint # 22

**Comment by jerome-regnier** [comment]
Vérification faite sur Checkbox item, Radio button item, et Switch item.
Le divider est hide par défaut. Ok Design
